meta_test.go

```diff
@@ -56,48 +56,173 @@ func (s *MetaSuite) TestReadMetaVersion2(c *gc.C) {
 	c.Assert(meta.Terms, gc.HasLen, 0)
 }
 
+func (s *MetaSuite) TestValidTermFormat(c *gc.C) {
+	valid := []string{
+		"foobar",
+		"foobar/27",
+		"foo/003",
+		"owner/Foo-bar",
+		"owner/foobar/27",
+		"owner/foobar",
+		"owner/foo-bar",
+		"owner/foo_bar",
+		"own-er/foobar",
+		"ibm/j9-jvm/2",
+		"term_123-23aAf/1",
+		"cs:foobar/27",
+	}
+
+	invalid := []string{
+		"/",
+		"/1",
+		"//",
+		"//2",
+		"27",
+		"owner/foo/foobar",
+		"@les/term/1",
+		"own_er/foobar",
```

Comment:
I'd think it would be good to have the actual error strings in the invalid tests, so we can see whether the error messages look reasonable.

Also, I'd suggest some more invalid forms (multiple colons, invalid tenants, single hyphens, etc)